ui/scss/component/_file-render.scss

```diff
@@ -411,111 +418,15 @@ $control-bar-icon-size: 0.8rem;
     pointer-events: none;
   }
 
-  // control-bar: time
   .vjs-control-bar {
-    .vjs-time-control {
-      line-height: $control-bar-height;
-      font-size: $control-bar-font-size;
-      display: flex;
-    }
-
-    .vjs-current-time {
-      padding-right: 0;
-      margin-right: 0.4rem;
-      @media (max-width: $breakpoint-small) {
-        margin-right: 0.2rem;
-      }
-    }
-
-    .vjs-time-divider {
-      min-width: unset;
-      padding: 0;
-      z-index: 0; // solves the grayed-out divider
+    .vjs-remaining-time {
+      display: none;
     }
 
+    .vjs-current-time,
+    .vjs-time-divider,
     .vjs-duration {
-      padding-left: 0;
-      margin-left: 0.4rem;
-      @media (max-width: $breakpoint-small) {
-        margin-left: 0.2rem;
-      }
-    }
-  }
-
-  // control-bar: menu, button and icon
-  .vjs-control-bar {
-    .vjs-menu-button {
-      font-size: $control-bar-font-size;
-      line-height: $control-bar-height;
-    }
-
-    .vjs-icon-placeholder {
-      font-size: $control-bar-icon-size; // videojs icon font
-      line-height: $control-bar-height;
-    }
-
-    .vjs-icon-placeholder::before {
-      line-height: $control-bar-height;
-    }
-
-    .vjs-quality-selector {
-      .vjs-icon-placeholder {
-        font-size: $control-bar-font-size;
-      }
-    }
-
-    .vjs-play-control,
-    .vjs-fullscreen-control {
-      .vjs-icon-placeholder {
-        // Compensate: these icons in Font VideoJs are smaller than their peers.
-        font-size: calc(#{$control-bar-icon-size} * 1.2);
-      }
-    }
-
-    .vjs-playback-rate {
-      .vjs-playback-rate-value {
-        font-size: $control-bar-font-size;
-        line-height: $control-bar-height;
-      }
-
-      .vjs-menu {
-        width: 10em; // Extend the width to prevent a potential scrollbar from blocking the text.
-        left: -3em; // Center the popup on top of the button that invoked it.
-      }
-    }
-
-    .vjs-chromecast-button {
-      .vjs-icon-placeholder {
-        $chromecast-icon-size: 21px;
-        width: $chromecast-icon-size;
-        height: $chromecast-icon-size;
-      }
-    }
```

Comment:
- chromecast button sizing lost (should roughly match the rest, I think)
- playback rate popup no longer center at the button (minor).

![image](https://user-images.githubusercontent.com/64950861/147683496-0ce31147-c0a7-474a-9f9c-1687cfb6bba6.png)

Reply:
Looks like the chromecast icon is distorting the position of the playback rate popup. In other words, issue 1 is causing issue 2. I have to find a way to trigger that icon on my end. I don't have a chromecast device.

Reply:
Fixed.

Reply:
Popup and button is now centered, but the icon still feels a bit too small compared to the reset.

![image](https://user-images.githubusercontent.com/64950861/147721860-aaab1094-2f6a-4017-ad4b-e8d1be74ac18.png)

Reply:
I'm now manually overriding the setting for this one button. Dunno what is causing the issue tho. Btw, some of these icons still look like they were randomly picked from a collection. I'd like to redesign all of them in a future update to make the player look more unified.
![image](https://user-images.githubusercontent.com/94839278/147735157-bd9aab59-5389-49ad-bfef-2d5c7926e317.png)